### ISSUE TITLE
Suite 4.7 Feedback on PostGIS Workshop

### ISSUE THREAD

**Issue by jodygarnett** [boundlessgeo/workshops#21]
TLDR: A few things to fix up, and a few things are hard to find.

Workshop materials:

* [ ] we link http://files.boundlessgeo.com/workshopmaterials/postgis-workshop-201401.zip - but I cannot find all the files that make up this zip in version control?

     Example where is sql workbook in version control?

* [ ] I am not sure where the slides that go with this presentation are stored

Workshop corrections:

* [ ] sql examples out of sync for geometry exercise (last three examples).
* [ ] correction: DROP INDEX nyc_census_blocks_geom_idx;
* [ ] QGIS strongly recommended for a visual sanity check on what is going on :)

Feedback and questions:

* Q: How to manage which grid-shift to use for NAD (US vs Canada)
  
  FAQ: http://proj.maptools.org/faq.html

* Q: How to fix table SRID when not specified in import?

  http://postgis.net/docs/UpdateGeometrySRID.html
  
  ```
  SELECT UpdateGeometrySRID('geometries','geom',4326);
  ```
  
  That is a lot easier than with PostGIS 1.x
